library/cmac.c

```diff
@@ -278,7 +278,7 @@ int mbedtls_cipher_cmac_update( mbedtls_cipher_context_t *ctx,
     }
 
     /* n is the number of blocks including any final partial block */
-    n = ( ilen + block_size - 1 ) / block_size;
+    n = (int)( ( ilen + block_size - 1 ) / block_size );
```

Comment:
I do not think this will overflow because of the division. However, I would suggest that instead of casting `n` to an `int` we declare `j` and `n` as `size_t` and change the for loop in line 284 to something like:
```
for( j = 1; j < n; j++ )
```